### PR TITLE
Updates for js readmes for November release

### PR DESCRIPTION
**Pull request by ramya-rao-a**
This PR updates the
- October release notes to remove the @next tags. It gives us the freedom to remove these tags on the GA released libraries without having to worry about people using the instruction here and getting the error that the tag doesnt exist
- November release notes to use the right version numbers and links

### CODE REVIEW

File: releases/2019-11/2019-11-js.md [hunk 1]
```diff
@@ -28,33 +28,32 @@ If you have a bug or feature request for one of the libraries, please post an is
...
--	API Improvements to the `EventProcessor` class that allows one to have distributed, load-balanced event processing.
-- The durable checkpoint store for the above can now support event hubs & consumer groups of the same name, but in different namespaces.
-- Removal of the feature that allowed one to consume events from an IotHub instance using an `EventHubClient` created using
-the IotHub connection string. The feature to do the same using an EventHub compatible IotHub connection string remains.
-- Distributed tracing support is added when sending events.
+- The library has undergone a major redesign where we now have separate clients for sending and receiving events.
+    - Use `EventHubProducerClient` to send events to Azure Event Hubs
+    - Use `EventHubConsumerClient` to receive events from Azure Event Hubs
+- The pull model for receiving events has been removed in favor of an unified push model that will work for:
+    - Receive from given single partition
```
Comment (bterlson) [inline releases/2019-11/2019-11-js.md]: nit: consider present participle for these bullets (receiving)

File: releases/2019-11/2019-11-js.md [hunk 2]
```diff
@@ -28,33 +28,32 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+| [Storage - File - Shares](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction) | [Package](https://www.npmjs.com/package/@azure/storage-file-share/v/12.0.0-preview.6) | [Readme.md](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file) | [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/storage.html#azure-storage-file-share) | [Changelog](https://github.com/Azure/azure-sdk-for-js/blob/%40azure/storage-file-share_12.0.0-preview.6/sdk/storage/storage-file-share/ChangeLog.md) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-file-share/samples) |
```
Comment (bterlson) [inline releases/2019-11/2019-11-js.md]: I'd say Storage - File Shares

File: releases/2019-10-11/2019-10-11-js-preview4.md [hunk 3]
```diff
@@ -49,12 +49,12 @@ More details are in the changelogs linked below.
...
+| [Storage - Files](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction) | [Package](https://www.npmjs.com/package/@azure/storage-file/v/12.0.0-preview.5) | [Readme.md](https://github.com/Azure/azure-sdk-for-js/tree/feature/storage/sdk/storage/storage-file) | [API Reference Documentation](https://azure.github.io/azure-sdk-for-js/storage-file/index.html) | [Changelog](https://github.com/Azure/azure-sdk-for-js/releases/tag/%40azure%2Fstorage-file_12.0.0-preview.5) | [Samples](https://github.com/Azure/azure-sdk-for-js/tree/feature/storage/sdk/storage/storage-blob/samples) |
```
Comment (bterlson) [inline releases/2019-10-11/2019-10-11-js-preview4.md]: Storage - File Shares?

Comment (ramya-rao-a) [reply releases/2019-10-11/2019-10-11-js-preview4.md]: Only for the November release. This is the October one, so keeping it as is.

File: releases/2019-11/2019-11-js.md (hunk 2 above)
Comment (ramya-rao-a) [reply releases/2019-11/2019-11-js.md]: fixed

File: releases/2019-11/2019-11-js.md (hunk 1 above)
Comment (ramya-rao-a) [reply releases/2019-11/2019-11-js.md]: fixed

File: releases/2019-11/2019-11-js.md (hunk 2 above)
Comment (ramya-rao-a) [reply releases/2019-11/2019-11-js.md]: fixed

**Pull request merged**